pkg/driver/node.go

```diff
@@ -73,12 +93,41 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 	// TODO when CreateVolume is implemented, it must use the same key names
 	subpath := "/"
 	encryptInTransit := true
+	iamAuth := false
```

Comment:
With regards to this, would it make more sense to feature flag this entire idea, then you can assume that if the user has turned the feature flag on this is what they want which gets rid of the need for this?

Reply:
Ok for the feature flag, because the CSIDriver object is immutable and we need to keep backward compatibility as you mentioned. We need `iamAuth`  there will be  cases where `iam auth` is not needed. The Driver is used across the cluster and you might have a mix of EFS that use IAM or not. If it is set to true, `efs-utils` behavior when provided with `iam` mount option but there is no `IAM` credential available will try to use whatever credential available including instance profile which is not something desirable.

Reply:
nit - suggest to change the var names here and later as well